samples/java-security-usage/k8s/deployment.yml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    app: java-security-usage
+    kyma-grafana: enabled
+  name: java-security-usage
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: java-security-usage
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: java-security-usage
+        kyma-grafana: enabled
+    spec:
+      containers:
+        - image: <placeholder for image repository>
+          imagePullPolicy: Always
+          name: java-security-usage
+          ports:
+            - name: http
+              containerPort: 8080
+          volumeMounts:
+            - name: xsuaa
+              mountPath: "/etc/secrets/sapcp/xsuaa/xsuaa-instance"
+              readOnly: true
+            - name: xsuaa2
+              mountPath: "/etc/secrets/sapcp/xsuaa/xsuaa-instance-1"
+              readOnly: true
+
+          envFrom:
+            - secretRef:
```

Comment:
what is meant with that optional secretRef?

Reply:
this is removed in later PR